multiqc/modules/bamutil/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Init for bamutil module
+from multiqc.modules.bamutil.clipoverlap import MultiqcModule
```

Comment:
If it's one tool with multiple subcommands, should have a `bamutil.py` file that imports each subcommand. See samtools and picard for examples.